code/processor/export_dados_bd.R

```diff
@@ -103,8 +103,19 @@ info_item_contrato <- import_itens_contrato(anos) %>%
 
 ## Alterações contratos
 message("#### alterações de contratos...")
-source(here("code/contratos/processa_alteracoes_contratos.R"))
-info_alteracoes_contrato <- processa_info_alteracoes_contratos(anos)
+source(here::here("code/contratos/processa_alteracoes_contratos.R"))
+source(here::here("code/contratos/processa_tipos_alteracao_contrato.R"))
+
+alteracoes <- import_alteracoes_contratos(anos) %>% 
+  processa_info_alteracoes_contratos()
+
+tipo_operacao_alteracao <- processa_tipos_alteracao_contrato()
+
+info_alteracoes_contrato <- alteracoes %>% 
+  join_alteracoes_contrato_e_tipo(tipo_operacao_alteracao) %>% 
+  join_alteracoes_contrato_e_contrato(info_contratos) %>% 
+  generate_id(TABELA_ALTERACOES_CONTRATO, ALTERACOES_CONTRATO_ID) %>% 
```

Comment:
Aqui dá o seguinte erro: Error in `[<-.data.frame`(`*tmp*`, , id, value = c("51", "50")) : 
  replacement has 2 rows, data has 0